src/sentry/api/endpoints/organization_member/details.py

```diff
@@ -241,9 +243,23 @@ def put(
             if not is_invite_from_user:
                 return Response({"detail": ERR_MEMBER_INVITE}, status=403)
 
+        logger = logging.getLogger(__name__)
+
         # XXX(dcramer): if/when this expands beyond reinvite we need to check
         # access level
         if result.get("reinvite"):
+            logger.info(
+                "organization.member_reinvite",
```

Comment:
if you want to check if the member details aren't being updated, don't you also need to know about the current role and teams for the member?